### PR TITLE
remove Travis CI

### PR DESCRIPTION
**Pull request by graingert**
### Contributor Checklist:

* [x] I have updated the release notes at `docs/source/NEWS.rst`
* [ ] I have updated the automated tests.
* [ ] All tests pass on your local dev environment. See `CONTRIBUTING.rst`.

### CODE REVIEW

**Review by adiroiban — COMMENTED** [review]
looks like this is remove travis-ci + remove python2

I guess that it will look ok later... looking forward for 20.0.0 release :)

**Comment by graingert** [comment]
@adiroiban yes, this PR is just to show I've not forgotten

File: docs/source/NEWS.rst [hunk 1]
```diff
@@ -5,6 +5,7 @@ Changelog
 -------------------
 
 - Dropped support for Python 2
+- removed Travis CI
```
Comment (adiroiban) [inline docs/source/NEWS.rst:8]: I see the NEWS file as something an external user will read in order to decide if they should update or now.

I don't think that we need put internal dev cleanup in the NEWS... but I am ok to merge it as it is.

**Review by adiroiban — APPROVED** [review]
All good. Thanks.
Only minor comments.

**Pull request merged**